src/print/Service.js

```diff
@@ -163,7 +163,8 @@ PrintService.prototype.createSpec = function(
     rotation: customAttributes.rotation
   });
 
-  this.goodnessOfFit_ = customAttributes['goodnessOfFit'];
+  this.goodnessOfFit_ = customAttributes.goodnessOfFit;
+  delete customAttributes.goodnessOfFit;
```

Comment:
This is a big hack, it shouldn't be passed throw the customAttributes, but throw a new optional? attribute.